### ISSUE TITLE
VCard attributes output as Arin.NET.Entities.ContactCardProperty

### ISSUE THREAD

**Issue by docfresh** [SteveSyfuhs/Arin.NET#2]
Cool project.  Noticed one thing. When outputting the vCard, the properties show up as the class name, not the actual value.

![image](https://user-images.githubusercontent.com/8064022/96809330-466bbd00-13e0-11eb-82c1-9ce166a1e928.png)

**Comment by SteveSyfuhs** [comment]
Yeah, the command line tool isn't really intended for production use or anything. More so just sanity checking. The underlying API exposes these fields correctly. You're welcome to make a PR to display it properly though.

________________________________
